boards/artix7_35t.json

```diff
@@ -1,31 +1,32 @@
 {
   "build": {
     "extra_flags": "-D ARTIX7_35T",
-    "f_cpu": "320000000L",
+    "f_cpu": "50000000L",
     "hwids": [
       [
-        "0x0403",
-        "0x6010"
+        "0x002a",
+        "0x15ba"
       ]
     ],
     "ldscript": "link.ld",
     "mabi": "ilp32",
     "march": "rv32imac",
     "mcmodel": "medany",
-    "mcu": "e31"
+    "mcu": "E-Class"
   },
   "debug": {
-    "onboard_tools": [
-      "ftdi"
-    ]
+
+	 "onboard_tools": ["ftdi"],
+         "svd_path":"arty35t.svd"
```

Comment:
Please use formatting that is consistent with other parts of the file: no empty lines, 2 whitespace characters for indentation.